src/styles/utils/sizing.ts

```diff
@@ -25,8 +25,8 @@ export default {
         width: '15%',
     },
 
-    w20: {
-        width: '20%',
+    w20px: {
```

Comment:
You might be able to just reuse our iconSizeNormal var? https://github.com/Expensify/App/blob/68d16de6be7d35851145283e892d1ee814d570d1/src/styles/variables.ts#L79

Reply:
Fixed!